platform/admin/core/admin-core-insecuredefaults/src/test/java/org/codice/ddf/admin/insecure/defaults/service/InsecureDefaultsCollectorTest.java

```diff
@@ -64,6 +69,30 @@ public void validate() {
     assertThat(details.iterator().next(), is(BASE_ALERT_TEST_MESSAGE + 0));
   }
 
+  @Test
+  public void validateUserPropertiesNotice() throws IOException {
+    // Setup
+    final int validatorCount = 3;
+    InsecureDefaultsCollector bean = createInsecureDefaultsServiceBean(validatorCount);
+    ArgumentCaptor<Event> captor = ArgumentCaptor.forClass(Event.class);
```

Comment:
❓  If you haven't run assertions against the captured values, do you really need this?

Reply:
Do you mean `captor`? If so I am using it a few lines down:
verify(eventAdmin, times(2)).postEvent(**captor**.capture());

Reply:
Yes because I use it a couple of lines below:
`verify(eventAdmin, times(2)).postEvent(captor.capture())`

Reply:
Seems like you're only capturing values. You're not calling `captor.getValues()` anywhere to run validation on the values, which would be the purpose of an argument captor. 

I would remove the captor and replace the capture calls with matchers such as `anyObject()` or `any(ClassYouExpect.class)`, for example.